vtex-apps/place-components
# Remove store builder and fix some errors

#### What problem is this solving?
This PR aims to fix some issues seen while implementing the Shipping step on Checkout v0. We also removed the `store` builder, as it was only used during the standalone development of this app and should be removed.

The main issues were seen with the `PlaceDetails` component (which didn't have a default value for the `display` prop, that caused it to throw an error) and with the `AddressForm` component (that didn't know how to handle addresses with a `receiverName` field). Some of the changes in this PR was just housekeeping and little refactors.

#### How should this be manually tested?
[Workspace](https://lucas--checkoutio.myvtex.com/checkout/#/shipping). The shipping step is still in development, but you can see the usages of this component.

I noticed that the `DeviceCoordinator` component isn't working properly, but that is an issue in the `places-graphql` app, not here.

#### Checklist/Reminders


#### Screenshots or example usage

<img width="651" alt="image" src="https://user-images.githubusercontent.com/10223856/79142528-07eea900-7d92-11ea-916f-a4d5f9509723.png">

#### Type of changes


✔️ | Type of Change
---|---
✔️ | Bug fix 
_ | New feature 
_ | Breaking change 
✔️ | Technical improvements
